hazelcast/src/test/java/com/hazelcast/cluster/ClusterJoinTest.java

```diff
@@ -52,6 +57,37 @@ public void killAllHazelcastInstances() throws IOException {
 
     @Test(timeout = 120000)
     @Category(ProblematicTest.class)
```

Comment:
You should not mark your test as `ProblematicTest`. This is just used for frequently failing tests which need further investigation. They are marked with this category so they don't mark a build as failed. Since your test is new we should not cover it, if it fails we should know.